streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/RackUtils.java

```diff
@@ -38,26 +40,37 @@ public final class RackUtils {
 
     private RackUtils() { }
 
-    public static Map<TopicPartition, Set<String>> getRacksForTopicPartition(final Cluster cluster,
-                                                                             final InternalTopicManager internalTopicManager,
-                                                                             final Set<TopicPartition> topicPartitions,
-                                                                             final boolean isChangelog) {
-        final Set<String> topicsToDescribe = new HashSet<>();
-        if (isChangelog) {
-            topicsToDescribe.addAll(topicPartitions.stream().map(TopicPartition::topic).collect(
-                Collectors.toSet()));
-        } else {
-            topicsToDescribe.addAll(topicsWithMissingMetadata(cluster, topicPartitions));
-        }
+    public static void annotateWithTopicPartitionsWithRackInfo(final Cluster cluster,
+                                                                    final InternalTopicManager internalTopicManager,
```

Comment:
nit: fix the formatting/indentation (although this will change when you change the method name anyway)